HISTORY.rst

```diff
@@ -6,11 +6,14 @@ History
 0.7.0 (unreleased)
 ++++++++++++++++++
 
+Features:
+
+* Refactor MozillaOIDCAuthenticationBackend so that the algorithm and verification key can be overridden in subclasses when you need to ...
```

Comment:
The class is called `OIDCAuthenticationBackend` and the sentence should probably mention that you can override the **method** for getting the key-for-verification. The algorithm you can already control with `settings.OIDC_RP_SIGN_ALGO`, `settings.OIDC_RP_IDP_SIGN_KEY` and `settings.OIDC_RP_CLIENT_SECRET`.

Reply:
nit: The sentence ends with a ` ...` instead of a regular `.`.